config/dataset_config.tsv

```diff
@@ -0,0 +1,17 @@
+dataset	EnhancerPredictionsAllPutative	EnhancerList	activity
```

Comment:
For checking in code, we'll want a small example config here (similar size to the test one you provided). (you'd keep your personalized config files locally).

Many people can look at this file as a template and substitute the values with their own